### DISCUSSION TITLE
Lamda function with presignedPutObject

### DISCUSSION THREAD

**Discussion by AaronPorts** [minio/minio#18823] [Q&A] [answered]
Is it possible to call a lambda function after an object has been loaded to a bucket with "PUT" request?

**Comment by klauspost** [comment, marked as answer]
You can set up [Bucket Notifications](https://min.io/docs/minio/linux/administration/monitoring/bucket-notifications.html#minio-bucket-notifications) to receive events when objects are uploaded.
